mast_aladin_lite/tests/test_overlays_dict.py

```diff
@@ -0,0 +1,404 @@
+from astropy.table import Table
+from astropy.coordinates import SkyCoord, Angle
+import astropy.units as u
+import pytest
+from unittest.mock import Mock
+import warnings
+import re
+
+from mast_aladin_lite import MastAladin
+from mast_aladin_lite.mast_overlay import MastOverlay, MastOverlayType
+from ipyaladin.elements.error_shape import EllipseError, CircleError
+from regions import CircleSkyRegion
+from ipyaladin import Marker
+
+
+mast_aladin = MastAladin()
+
+
+def test_overlays_dict_add_markers(
+    monkeypatch,
+):
+    """Test overlays_dict overlay info from adding markers."""
+    test_name = "test"
+    mock_send = Mock()
+    monkeypatch.setattr(MastAladin, "send", mock_send)
```

Comment:
suggestion: When patching things like send, I really like the `@patch` decorator for readability and ease of use. Here, I believe it would look like the following
```suggestion
from unittest.mock import patch

@patch('MastAladin.send')
def test_overlays_dict_add_markers(mock_send):
    """Test overlays_dict overlay info from adding markers."""
```
patching this way is weird because you have to ensure that you're mocking the right path, so if you try this and it doesn't work, lmk and I have some other suggestions that might have the path correct. 

another thing to note, that isn't important here. If you have `@patch` to mock multiple functions, the order of those mocks is reversed in the function args
```
@patch('funcA')
@patch('funcB')
def test_my_functions(mock_func_b, mock_func_a)
```